src/themes/charts/themes.ts

```diff
@@ -38,37 +38,75 @@ import {
 
 import { RecursivePartial } from '../../components/common';
 
-// @ts-ignore typescript doesn't understand the webpack loader
-import lightColors from '!!sass-vars-to-js-loader!../../global_styling/variables/_colors.scss';
-// @ts-ignore typescript doesn't understand the webpack loader
-import darkColors from '!!sass-vars-to-js-loader!../../themes/oui/oui_colors_dark.scss';
+// @ts-ignore typescript doesn't understand the babel preprocessor
+import {
+  ouiColorChartBand as ouiV7ColorChartBand,
+  ouiColorChartLines as ouiV7ColorChartLines,
+  ouiColorDarkShade as ouiV7ColorDarkShade,
+  ouiColorDarkestShade as ouiV7ColorDarkestShade,
+  ouiColorEmptyShade as ouiV7ColorEmptyShade,
+  // @ts-ignore typescript doesn't understand the babel preprocessor
+} from '!!variables-from-scss!!../../global_styling/variables/_colors.scss';
+
+// @ts-ignore typescript doesn't understand the babel preprocessor
+import {
+  ouiColorChartBand as ouiV7ColorChartBandDark,
+  ouiColorChartLines as ouiV7ColorChartLinesDark,
+  ouiColorDarkShade as ouiV7ColorDarkShadeDark,
+  ouiColorDarkestShade as ouiV7ColorDarkestShadeDark,
+  ouiColorEmptyShade as ouiV7ColorEmptyShadeDark,
+  // @ts-ignore typescript doesn't understand the babel preprocessor
+} from '!!variables-from-scss!!../../themes/oui/oui_colors_dark.scss';
+
+const ouiV7FontFamily =
```

Comment:
Is there a variable we can extract this from?

Reply:
There is but it wasn't worth the effort since we will be getting rid of `charts`